cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -647,6 +646,15 @@ def check_deployments(manager, old_deployments, logger,
     _log('Found correct deployments', logger, tenant)
 
 
+def check_status(manager):
+    status = manager.client.manager.get_status()
+    for service in status['services']:
+        for instance in service['instances']:
+            if instance['state'] != 'running':
+                raise Exception('Service %s is in status %s',
```

Comment:
this needs to be formatted here, because `Exception` won't do it automatically
like for example:
```python
raise Exception('Service {0} is in status {1}'.format(instance, instance['state']))
```

Reply:
Fixed